client/src/pages/GetTasksFiltered/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { useState, useEffect } from 'react'
+import { Task } from '../../types/taskTypes'
+import { getTasksByCriteria } from '../../services/taskService'
+import './GetTasksFiltered.css'
+
+const GetTasksFiltered = () => {
+  const [tasks, setTasks] = useState<Task[]>([])
+  const [tags, setTags] = useState<string>('')
+  const [search, setSearch] = useState<string>('')
+
+  const loadTasksByCriteria = async () => {
+    try {
+      const fetchedTasks = await getTasksByCriteria({ tags, search })
+      setTasks(fetchedTasks)
+    } catch (error) {
+      console.log('Error fetching tasks: ', error)
+    }
+  }
+
+  useEffect(() => {
+    loadTasksByCriteria()
+  }, [tags, search])
```

Comment:
In your code, you will be sending many API calls. To prevent excessive API calls when the user types in the search or tags input, you can implement a debounce mechanism.
```javascript
  import { debounce } from 'lodash'
  const debouncedLoadTasks = debounce(loadTasksByCriteria, 300)

  useEffect(() => {
    debouncedLoadTasks(tags, search)
  }, [tags, search])
```

Reply:
Nice one! I like it!

Reply:
it's done